code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -61,13 +61,17 @@
 				/obj/item/bodybag = 10,
 				/obj/item/clothing/glasses/meson = 20,
 				/obj/item/clothing/glasses/sunglasses = 10,
-				/obj/item/clothing/gloves/color/yellow/fake = 10,
+				/obj/item/clothing/gloves/color/yellow/fake = 15,
+				/obj/item/clothing/gloves/color/fyellow = 10,
+				/obj/item/clothing/gloves/color/yellow = 5,
+				/obj/item/clothing/gloves/color/black = 20,
+				/obj/machinery/portable_atmospherics/hydroponics = 10,
```

Comment:
yes you can find full trays in maint now

though they spawn wrenched not sure if a problem